go-controller/pkg/ovn/master_test.go

```diff
@@ -409,13 +427,17 @@ var _ = Describe("Master Operations", func() {
 			f, err = factory.NewWatchFactory(fakeClient)
 			Expect(err).NotTo(HaveOccurred())
 
-			clusterController := NewOvnController(fakeClient, f, stopChan, newFakeAddressSetFactory())
+			clusterController := NewOvnController(fakeClient, f, stopChan,
+				newFakeAddressSetFactory(), mockOVNNBClient,
+				mockOVNSBClient)
+
 			Expect(clusterController).NotTo(BeNil())
 			clusterController.TCPLoadBalancerUUID = tcpLBUUID
 			clusterController.UDPLoadBalancerUUID = udpLBUUID
 			clusterController.SCTPLoadBalancerUUID = ""
 
 			err = clusterController.StartClusterMaster("master")
+			klog.Errorf("err: %v", err)
```

Comment:
leftover line?

Reply:
Perhaps. But it's useful to log an error if the `StartClusterMaster` returns an error. The ginkgo framework isn't the best when it comes to logging errors and dumping stacktrace in every single failure. There are times when running the tests causes failures, but it doesn't print any discernible errors.

Reply:
@abhat I would remove the klog. If Ginkgo does not print the error out as part of the Expect(), then the problem is elsewhere and not in this testcase. We should fix it in that other place. But I'm struggling to think of when this wouldn't trigger, usually it's Expects() deeper down in other goroutines.

Reply:
@abhat last question; why only here? if we need a print here, why not everywhere else StartClusterMaster() gets called?

Reply:
left over from testing. deleted it.